code/Aspects/CacheAfterCallAspect.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LeKoala\DebugBar\Aspect;
+
+use LeKoala\DebugBar\Collector\SilverStripeCollector;
+use Monolog\Logger;
+use SilverStripe\Core\Injector\AfterCallAspect;
+use SilverStripe\Core\Injector\Injector;
+
+class CacheAfterCallAspect implements AfterCallAspect
+{
+    /**
+     * Logs all hits/misses after a CacheInterface::get call is made.
+     *
+     * {@inheritdoc}
+     */
+    public function afterCall($proxied, $method, $args, $result)
+    {
+        $message = "Your partial cache named {$args[0]} ";
+        $message .= (empty($result)) ? 'didn’t find any existing caches' : 'found an existing cache';
+        SilverStripeCollector::addTemplateCacheInfo($message);
```

Comment:
I feel like this method should be handling structured data rather than formatting. The SilverStripeCollector shouldn't necessarily be the place that has this pushed to it, but should be the area that asks for it - I'm not saying the SSViewer proxy is the best example, but it stores the template data on itself and the collector pulls it in.

I'm also not sure whether we need the "Your partial cache named" etc messages - not sure what the best way to display it otherwise might be, but perhaps in a list or something:

* 💚  cache_key_that_was_hit
* 🔴  cache_key_that_was_missed

P.s. please don't use those emojis 😆